### PR TITLE
kappa day 4

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+author: thor_schaeff
+image: launch-week-8/day-4/integration-marketplace-og.jpg
+thumb: launch-week-8/day-4/integration-marketplace-thumb.jpg
+---
+
+We've been running our [Integrations Marketplace](/partners) in “stealth mode” for about a year now. What started as a dog-fooding project has now transformed into a marketplace with [over 60 integrations](/partners/integrations). (It's also an [open source template](https://vercel.com/templates/next.js/supabase-partner-gallery) that you can use yourself).
+
+![Featured](/images/blog/launch-week-8/day-4/featured-integrations.png)
+
+Supabase Integrations allows Partners to extend the Supabase platform with useful tooling. Today we're adding [OAuth2.0 Applications](https://supabase.com/docs/guides/platform/oauth-apps/publish-an-oauth-app). For Supabase users, this makes it even easier to connect your favorite tools to the Supabase projects. Within minutes you can:
```
Comment (thorwebdev) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:20]:
```suggestion
Supabase Integrations allows Partners to extend the Supabase platform with useful tooling. Today we're adding [OAuth2.0 Applications](https://supabase.com/docs/guides/platform/oauth-apps/publish-an-oauth-app). For Supabase users, this makes it even easier to connect their favorite tools to their Supabase projects. Within minutes you can:
```

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+- Add your favorite [Low Code](https://supabase.com/partners/integrations#low-code) tools on top of your Supabase database.
+- Integrate your favorite [DevTools](https://supabase.com/partners/integrations#devtools): including secrets managers and database management tools.
+- Add [caching](https://supabase.com/partners/integrations#caching%20/%20offline-first) to your Supabase database.
+- Don't link the Supabase admin dashboard? Try [one of these](https://supabase.com/partners/integrations#data%20platform).
```
Comment (thorwebdev) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:25]:
```suggestion
- Not a fan of the Supabase admin dashboard? Try [one of these](https://supabase.com/partners/integrations#data%20platform).
```

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- Try out a different [SMS and email provider](https://supabase.com/partners/integrations#messaging).
+
+## Featured Partners
+
+For the initial launch we're started with a few development partners who we worked with to build and test the OAuth functionality.
```
Comment (thorwebdev) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:30]:
```suggestion
For the initial launch we've started with a few partners to help us build and test the OAuth functionality.
```

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+Read more on [Resend's blog](https://resend.com/blog/how-to-configure-supabase-to-send-emails-from-your-domain).
+
+### Snaplet
+
+![Snaplet x Supabase](/images/blog/launch-week-8/day-4/marketplace-snaplet.jpg)
+
+Snaplet is a tool for Typescript developers to copy your database, transform sensitive data, and share it with your team without worrying about PII. If you followed our [Tuesday launch](https://supabase.com/blog/supabase-local-dev#database-seeding) you'll be familiar with Snaplet - they are one of the best tools for [generating seed data](https://supabase.com/docs/guides/cli/seeding-your-database#generating-seed-data) for your local development environment. Now they are making it even easier with their official OAuth App build production-like development environments for you team.
```
Comment (thorwebdev) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:52]:
```suggestion
Snaplet is a tool for Typescript developers to copy your database, transform sensitive data, and share it with your team without worrying about PII. If you followed our [Tuesday launch](https://supabase.com/blog/supabase-local-dev#database-seeding) you'll be familiar with Snaplet - they are one of the best tools for [generating seed data](https://supabase.com/docs/guides/cli/seeding-your-database#generating-seed-data) for your local development environment. Now they are making it even easier, with their official OAuth App, to spin up production-like development environments for your team.
```

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+One that requires no introduction - since so many of you use Vercel, we've dedicated an entire blog post to the upgraded Vercel integration.
+
+Learn more about the Vercel integration [updates we're launching](/blog/using-supabase-with-vercel) today.
+
+### Windmill
+
+![Windmill x Supabase](/images/blog/launch-week-8/day-4/marketplace-windmill.jpg)
+
+[Windmill](https://windmill.dev) (YC [S22](https://www.ycombinator.com/companies/windmill)) is an open-source open source alternative to Retool and a modern Airflow. They provide a developer platform to quickly build production-grade complex workflows and integrations from minimal Python and Typescript scripts. Their one-click integration with Supabase makes it simple to launch new databases, process large quantities of data (maybe even convert them into [embeddings](https://supabase.com/vector)), and build internal dashboards.
```
Comment (thorwebdev) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:76]:
```suggestion
[Windmill](https://windmill.dev) (YC [S22](https://www.ycombinator.com/companies/windmill)) is an open source alternative to Retool and a modern Airflow. They provide a developer platform to quickly build production-grade complex workflows and integrations from minimal Python and Typescript scripts. Their one-click integration with Supabase makes it simple to launch new databases, process large quantities of data (maybe even convert them into [embeddings](https://supabase.com/vector)), and build internal dashboards.
```

**Review by thorwebdev — COMMENTED** [review]
Small nit fixes

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 6]
```diff
@@ -0,0 +1,137 @@
+---
+title: 'Supabase Integrations Marketplace'
```
Comment (awalias) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:2]: let's go with:
"Supabase Integrations" 
or
"Supabase Integrations v2.0" 
@kiwicopple wyt?

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+description: 'Become a Supabase Integrations Partner: Publish a OAuth Apps and Build with Supabase.'
```
Comment (awalias) [inline apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:3]:
```suggestion
description: 'Become a Supabase Integrations Partner: Publish OAuth Apps and Build with Supabase.'
```

File: apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx (hunk 6 above)
Comment (awalias) [reply apps/www/_blog/2023-08-10-supabase-integrations-marketplace.mdx:2]: or maybe "Supabase Integrations v1.0" based on the first sentence "we've been running our [Integrations Marketplace](/partners) in “stealth mode” for about a year now"

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**